shopfloor/tests/test_checkout_scan.py

```diff
@@ -89,20 +89,18 @@ def _test_scan_document_error_different_picking_type(self, barcode_func):
         self.assert_response(
             response,
             next_state="select_document",
-            message={
-                "message_type": "error",
-                "message": "You cannot move this using this menu.",
-            },
+            message={"message_type": "error", "message": msg},
         )
 
     def test_scan_document_error_different_picking_type_picking(self):
         self._test_scan_document_error_different_picking_type(
-            lambda picking: picking.name
+            lambda picking: picking.name, msg="You cannot move this using this menu."
         )
 
     def test_scan_document_error_different_picking_type_package(self):
         self._test_scan_document_error_different_picking_type(
-            lambda picking: picking.move_line_ids.package_id.name
+            lambda picking: picking.move_line_ids.package_id.name,
+            msg="Barcode not found",
```

Comment:
This is actually not the message we expect here, we should have "You cannot move this using this menu.", but it will be correct when I apply https://github.com/camptocamp/wms/pull/8/commits/92a58e6ea608e0354b7adc1a7be568f38f1ec924